test/elixir_bench/benchmarks/benchmarks_test.exs

```diff
@@ -1,18 +1,158 @@
 defmodule ElixirBench.BenchmarksTest do
   use ElixirBench.DataCase
-  alias ElixirBench.{Benchmarks, Benchmarks.Job}
-  alias ElixirBench.Repos
+  alias ElixirBench.{Benchmarks, Benchmarks.Job, Benchmarks.Runner}
+
+  import ElixirBenchWeb.TestHelpers
+  import ElixirBench.Factory
+
+  describe "create_runner/1" do
+    test "insert new Runner given correct params" do
+      assert_difference(Runner, 1) do
+        Benchmarks.create_runner(%{name: "myrunner", api_key: "sudo"})
+      end
+    end
+
+    test "return the new Runner inserted" do
+      name = "myrunner"
+      api_key = "sudo"
+
+      {:ok, runner} = Benchmarks.create_runner(%{name: name, api_key: api_key})
+
+      assert runner.id
+      assert %Runner{name: ^name, api_key: ^api_key} = runner
```

Comment:
I think this test might be enough for the runner tests - `create_runner` just creates a changeset and tries to insert it. All the ways in which that could or could not go wrong is probably better tested in the runner unit tests by just checking if the changeset is valid.

That tests the core component this relies on, is faster and more focussed :)